backend/test/v2/resources/hello-world.yaml

```diff
@@ -11,7 +11,7 @@ deploymentSpec:
         - hello world
         command:
         - echo
-        image: public.ecr.aws/docker/library/python:3.12
+        image: busybox:1.36
```

Comment:
No https://github.com/kubeflow/manifests/blob/1c74b37c023f93996201f8e703d89f47b9ab336a/.github/workflows/pipeline_swfs_test.yaml#L30 is the right way!